app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
```

Comment:
- [ ] The `comment` belong to the `author` and post, kindly make sure to set up associations for this so it works properly